<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerManager</class>
 <widget class="QWidget" name="ServerManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="cbSearchable">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>是否可被&quot;搜索&quot;到</string>
   </property>
   <property name="text">
    <string>可被搜索？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbChooseDirectory">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>浏览目录</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leServerDirectory">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>一般不能修改</string>
   </property>
   <property name="placeholderText">
    <string>服务器目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOpenMonitor">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>管理共享服务器</string>
   </property>
   <property name="text">
    <string>打开管理器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStart">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbStop">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRestart">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>重启</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lbTip">
      <property name="text">
       <string>服务器状态:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="leStatus">
      <property name="text">
       <string>运行中..</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="cbDiscoverable">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>是否可被&quot;发现&quot;到</string>
   </property>
   <property name="text">
    <string>可被发现？</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
